pkg/kubelet/winstats/winstats_test.go

```diff
@@ -123,6 +130,21 @@ func TestWinVersionInfo(t *testing.T) {
 		KernelVersion: "v42"})
 }
 
+func TestConvertCPUValue(t *testing.T) {
+	var cpuValue uint64 = 10000
```

Comment:
cpuValue is a percent so it's probably more realistic for this to be between 0 and 100. 

For this and below, can you also test some edge cases like 0 and 100, and some in-between value?

Reply:
Also recommend using subtests for each test case: https://blog.golang.org/subtests

Reply:
Done.